### ISSUE TITLE
App bar scrolls when it shouldn't

### ISSUE THREAD

**Issue by domdomegg** [commons-app/apps-android-commons#2165]
**Summary:** 

The toolbar scrolls when really it should be the contents of certain activities that scroll only. See GIF below demonstrating behaviour.

Most likely you will need to change files in `/res/layout/`

Places where this happens
- Achievements
- About
- other places I've forgotten to check?

Comment if you are working on this issue beforehand so we don't have two people working on the same thing. Feel free to submit one pull request that contains all changes, or many small ones. If it turns out to be kinda complex, go down the many small ones route. Do tag me (@domdomegg) if you need a review.

**Steps to reproduce:**

Open "About" activity and scroll up and down

**Screen-shots:** 

| Actual | Expected (toolbar stays fixed) |
| - | - |
| ![actual](https://user-images.githubusercontent.com/4953590/50188429-93306980-0319-11e9-89c7-d3fd64bb1bb7.gif) | ![expected](https://user-images.githubusercontent.com/4953590/50188431-94fa2d00-0319-11e9-9808-ea08baa6e2c8.gif) |

**Comment by vanshikaarora** [comment]
I am wiling to work on this error. Before that can you please elaborate on what phone you got this issue. Because I tried to do the same on my phone and i couldn't come across any error.

**Comment by neslihanturan** [comment]
Thanks @vanshikaarora , the issue is toolbar is also moving with page content. Ideally it should stay and content should scroll.

**Comment by vanshikaarora** [comment]
Sir i got what you have posted but in my case. For about actitvity neither the toolbar nor the activity is scrolled.

**Comment by domdomegg** [comment]
It's likely that your phone has a higher dp resolution so you can see the whole page without scrolling. Maybe try an emulator with a lower resolution or bumping up the display scaling on your phone?

**Comment by vanshikaarora** [comment]
OK sir I'll check this out on an emulator and then start working upon this.

**Comment by neslihanturan** [comment]
Thanks @vanshikaarora , and our team has a motto: No sirs here, just friends:)

**Comment by vanshikaarora** [comment]
Sorry to say that @neslihanturan. And thanks for that courtesy :)

**Comment by vanshikaarora** [comment]
Hey I am working upon this issue

**Comment by neslihanturan** [comment]
Thanks @vanshikaarora , did you managed to reproduce the issue?

**Comment by vanshikaarora** [comment]
No, I couldn't. I tried on an emulator but my system cannot suffered frequent non responsive states due to  gradle build of the heavy project as well as the emulator.

But yet i went through the xml file for these activities and i could get what was wrong with the code that generated this issue.

**Comment by neslihanturan** [comment]
Oh okay, if you have a theory go for it. But what about finishing your open PR first? It is always better working on one PR at at time.

**Comment by vanshikaarora** [comment]
@neslihanturan Now I'll start working upon this
